typebackend/socialconnectviews.py

```diff
@@ -0,0 +1,18 @@
+# from allauth.socialaccount.providers.facebook.views import FacebookOAuth2Adapter
+# from allauth.socialaccount.providers.github.views import GitHubOAuth2Adapter
+# from allauth.socialaccount.providers.twitter.views import TwitterOAuthAdapter
+# from allauth.socialaccount.providers.oauth2.client import OAuth2Client
+# from rest_auth.registration.views import SocialConnectView
+# from rest_auth.social_serializers import TwitterConnectSerializer
+
+# class FacebookConnect(SocialConnectView):
+#     adapter_class = FacebookOAuth2Adapter
+
```

Comment:
Is this file needed?

Reply:
No.i mean if we want we can use it for the user to connect using other social media login too